tensorboard/plugins/graphs/tf_graph_loader/tf-graph-loader.html

```diff
@@ -61,6 +61,7 @@
       readOnly: true,
       notify: true
     },
+    /** @type {Object?} */
```

Comment:
This is peculiar because Object should be nullable by default. Is the warning still there if you get rid of the `?`?